src/qml/qgismobileapp.qml

```diff
@@ -4082,6 +4082,10 @@ ApplicationWindow {
     codeReader: codeReader
     featureModel.currentLayer: dashBoard.activeLayer
 
+    onRequestJumpToPoint: function (center, scale, handleMargins) {
+      mapCanvasMap.jumpTo(center, scale, -1, handleMargins);
+    }
```

Comment:
You'll need to add the same slot in OverlayerFeatureFormDrawer, _and_ you'll need to add it in EmbeddedFeatureForm (with the necessary ties all the way to qgismobileapp). That's the last change, we're there after. Woupidou.